internal/services/v1/errors.go

```diff
@@ -269,7 +269,10 @@ func rewriteError(ctx context.Context, err error) error {
 		return status.Errorf(codes.Internal, "internal error: %s", err)
 	case errors.As(err, &graph.ErrUnimplemented{}):
 		return status.Errorf(codes.Unimplemented, "%s", err)
-
+	case errors.Is(err, context.DeadlineExceeded):
```

Comment:
Can we add a unit test for this?

Reply:
done in https://github.com/authzed/spicedb/pull/986/commits/0c08ff8669e98ad679e6dd07023764d0f8063b4c